{"name": "test_003_for_invalid_data[input1]", "status": "failed", "statusDetails": {"message": "AssertionError: [FAILURE] Actual Status code - 200 \nExpected status code \">299\" as input is invalid \nInput - {'dateofbirth': '1900-02-26', 'unit': 'days'}", "trace": "self = <Test.test_001.Test_001 object at 0x00000154786EDCF0>, input = {'dateofbirth': '1900-02-26', 'unit': 'days'}\n\n    @pytest.mark.parametrize(\"input\", d['invalid_data'])\n    def test_003_for_invalid_data(self, input):\n>       self.clientUtil.verify_invalid_data(params=input)\n\nTest\\test_001.py:26: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <utils.client.Client object at 0x00000154786EE2C0>, fail_on_error = True\nkwargs = {'params': {'dateofbirth': '1900-02-26', 'unit': 'days'}}, uri = '/Prod/next-birthday'\nresponse = <Response [200]>\n\n    def verify_invalid_data(self, fail_on_error=True, **kwargs):\n        uri = '/Prod/next-birthday'\n        response = RequestsUtils.get(host=self.base_url, path=uri, headers=self.header, **kwargs,\n                                      fail_on_error=fail_on_error)\n>       assert response.status_code > 299, f'[FAILURE] Actual Status code - {response.status_code} \\n' \\\n                                           f'Expected status code \">299\" as input is invalid \\n' \\\n                                           f'Input - {kwargs[\"params\"]}'\nE       AssertionError: [FAILURE] Actual Status code - 200 \nE       Expected status code \">299\" as input is invalid \nE       Input - {'dateofbirth': '1900-02-26', 'unit': 'days'}\n\nutils\\client.py:30: AssertionError"}, "attachments": [{"name": "log", "source": "a165605e-14d4-4a60-b5c4-dde446c5a4de-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "input", "value": "{'dateofbirth': '1900-02-26', 'unit': 'days'}"}], "start": 1685017229848, "stop": 1685017231324, "uuid": "76c48bad-ef19-4257-9a1c-68843752d48d", "historyId": "50d7bd46e5a5b005531e00cbdc1f4313", "testCaseId": "b377320e270af1a2d8e696e34b891a70", "fullName": "Test.test_001.Test_001#test_003_for_invalid_data", "labels": [{"name": "parentSuite", "value": "Test"}, {"name": "suite", "value": "test_001"}, {"name": "subSuite", "value": "Test_001"}, {"name": "host", "value": "DESKTOP-FEG7QCL"}, {"name": "thread", "value": "3360-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "Test.test_001"}]}